crates/core/src/oracle/multilinear.rs

```diff
@@ -601,15 +620,16 @@ impl DeserializeBytes for MultilinearPolyVariant<BinaryField128b> {
 		Ok(match u8::deserialize(&mut buf, mode)? {
 			0 => Self::Committed,
 			1 => Self::Transparent(DeserializeBytes::deserialize(buf, mode)?),
-			2 => Self::Repeating {
+			2 => Self::Transparent2(DeserializeBytes::deserialize(buf, mode)?),
```

Comment:
Please rename to `Structured`

Reply:
Dang, I really hoped to call it Transparent2!

Will do.